format.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env nix-shell
+#!nix-shell shell.nix -i bash --pure
+#
+# Format linear-base using pinned version of Ormolu.
+
+set -e
+
+export LANG="C.UTF-8"
+
+cabal format
+
+ormolu -m inplace $(find src -type f -name "*.hs-boot" -o -name "*.hs")
+ormolu -m inplace $(find test -type f -name "*.hs-boot" -o -name "*.hs")
+ormolu -m inplace $(find examples -type f -name "*.hs-boot" -o -name "*.hs")
+ormolu -m inplace $(find bench -type f -name "*.hs-boot" -o -name "*.hs")
```

Comment:
Any reason why this isn't 
```suggestion
ormolu -m inplace $(find . -type f -name "*.hs-boot" -o -name "*.hs")
```
If so, please document.

Reply:
In case there is a good reason to list all these directories, it's probably better to do
```suggestion
ormolu -m inplace $(find src test example bench -type f -name "*.hs-boot" -o -name "*.hs")
```

Reply:
There is no reason not to do what you suggested